tests/dist/test_dist_objects.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
This script mainly for launching servers and clients so it's ok to test `DistTensor`, `DistEmbedding` , `DistOptmizer` via it, even with `DistGraph`, `DistDataLoader`. But for now, how about focusing on `DistTensor` only? It's much clearer and we could extend it in the near future.